open-ce/docker_build.py

```diff
@@ -15,7 +15,7 @@
 HOME_PATH = "/home/builder"
 
 REPO_NAME = "open-ce"
-IMAGE_NAME = "open-ce-builder"
+IMAGE_NAME = "open-ce-open-ce"
```

Comment:
Maybe this should still be builder?

Reply:
Thanks, I tried to only replace the occurrences of builder that needed to be replaced, but I must have missed this one.